<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Peter Ralph">
  <title>Landscapes in population genetics:</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reset.css">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
  </style>
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/theme/simple.css" id="theme">
  <style type="text/css">

  .reveal { font-size: 30px; }

  .reveal h1 { font-size: 1.5em; } 

  .reveal h2 { font-size: 1.2em; } 

  .reveal .slides { text-align: left; }

  .reveal .slides figure { text-align: center; }

  .centered p { text-align: center; }

  .caption p { font-size: 75%; }

  div.floatright { float: right; text-align: right; }

  .reveal figcaption { display: none; }

  .reveal img { border: none; }

  .reveal section img { border: none; }

  .MathJax_Display { margin: 0; }

  img.finger { vertical-align: text-top; }

  </style>

  <div style="display: none">
  \[
  \newcommand{\E}{\mathbb{E}}
  \renewcommand{\P}{\mathbb{P}}
  \DeclareMathOperator{\var}{var}
  \]
  </div>
</head>
<body>
  <div class="reveal">
    <div class="slides">

<section id="title-slide">
  <h1 class="title">Landscapes in population genetics:</h1>
  <p class="subtitle">ecology, evolution, conservation, and simulation</p>
  <p class="author">Peter Ralph</p>
  <p class="date">Program in Ecology, Evolution &amp; Conservation Biology <br/> UIUC // 3 February 2021</p>
</section>

<section>
<section id="outline" class="title-slide slide level1">
<h1>Outline</h1>

</section>
<section id="outline-of-the-talk" class="slide level2">
<h2>Outline of the talk</h2>
<ol type="1">
<li>Big picture</li>
<li>Tools</li>
<li>Applications</li>
</ol>
</section></section>
<section>
<section id="adaptation-and-genetic-variation" class="title-slide slide level1">
<h1>Adaptation, and genetic variation</h1>
<!--
## Sickle-cell (HbS) allele frequencies

![sickle-cell HbS allele map, from Flint et al 1998](figs/HbS.png)


Human sickle-cell allele (HbS):
(Currat et al 2002)

- Single base substitution
- provides protection against malaria 
  (but deleterious in homozygotes)
-->
</section>
<section id="g6pd-deficiency-allele-frequencies" class="slide level2">
<h2>G6PD deficiency allele frequencies</h2>
<figure>
<img data-src="figs/g6pd-freqs.png" alt="glucose-6-phosphate dehydrogenase deficiency alleles from Howes et al 2013" /><figcaption aria-hidden="true">glucose-6-phosphate dehydrogenase deficiency alleles from Howes et al 2013</figcaption>
</figure>
<p>Human G6PD variants <em>(Howes et al 2013)</em></p>
<ul>
<li>over 130 G6PD deficiency alleles; 34 variants at high frequency</li>
<li>provide protection against malaria but increase risk of anemia</li>
<li>Estimated ages 40-400 generations</li>
</ul>
</section>
<section id="section" class="slide level2">
<h2></h2>
<figure>
<img data-src="figs/volcanic-outcrops-plus-mice.png" style="width:70.0%" alt="volcanic outcrops: mice by AH Harris" /><figcaption aria-hidden="true">volcanic outcrops: mice by AH Harris</figcaption>
</figure>
<ul>
<li>Dark-pigmented mammals and reptiles on volcanic outcrops in the Southwest. (Dice, Benson 1936)</li>
<li>‘Dark’ allele beneficial on outcrops, deleterious elsewhere.</li>
<li>MC1R: basis is shared <em>between species</em> but not <em>between populations</em> (Nachman, Hoekstra)</li>
</ul>
</section>
<section id="isolation-by-distance" class="slide level2">
<h2>“isolation by distance”</h2>
<figure>
<img data-src="figs/spacetree_1.png" alt="trees in space, by CJ Battey" /><figcaption aria-hidden="true">trees in space, by CJ Battey</figcaption>
</figure>
<div class="caption">
<p>by <a href="cjbattey.com">CJ Battey</a></p>
</div>
</section>
<section id="section-1" class="slide level2">
<h2></h2>
<figure>
<img data-src="figs/everyone-ibd.png" alt="genetic and geographic distance for desert tortoises" /><figcaption aria-hidden="true">genetic and geographic distance for desert tortoises</figcaption>
</figure>
<ul>
<li>genetic versus geographic distance between pairs of 272 desert tortoises (McCartney-Melstad, Shaffer)</li>
<li>clouds are comparisons within/between the two colors</li>
</ul>
</section>
<section id="section-2" class="slide level2">
<h2></h2>
<p>How much of the genome is under selection?</p>
</section></section>
<section>
<section id="genomic-landscapes" class="title-slide slide level1">
<h1>Genomic landscapes</h1>

</section>
<section id="section-3" class="slide level2">
<h2></h2>
<figure>
<img data-src="figs/from_the_literature/langley-et-al-2012-chr3-pi-and-rho.png" alt="Langley et al 2012" /><figcaption aria-hidden="true">Langley et al 2012</figcaption>
</figure>
</section>
<section id="diversity-correlates-with-recombination-rate" class="slide level2">
<h2>Diversity correlates with recombination rate</h2>
<div class="columns">
<div class="column" style="width:80%;">
<figure>
<img data-src="figs/from_the_literature/corbett-detig-divergence-recomb-all-species.png" style="width:85.0%" alt="Corbett-Detig et al" /><figcaption aria-hidden="true">Corbett-Detig et al</figcaption>
</figure>
</div><div class="column" style="width:20%;">
<p><em>Hudson 1994; Cutter &amp; Payseur 2013; Corbett-Detig et al 2015</em></p>
</div>
</div>
</section>
<section class="slide level2">

<dl>
<dt>linked selection</dt>
<dd><p>The indirect effects of selection on genomic locations that are <em>linked</em> to the sites under selection by a lack of recombination.</p>
</dd>
</dl>
<div class="centered">
<p><img data-src="figs/sweep_haps.png" class="fragment" style="width:40.0%" /> <img data-src="figs/background_haps.png" class="fragment" style="width:40.0%" /></p>
</div>
</section>
<section id="the-mimulus-aurantiacus-species-complex" class="slide level2">
<h2>The <em>Mimulus aurantiacus</em> species complex</h2>
<div class="centered">
<p><img data-src="figs/aurantiacus/phylogeny.png" /></p>
</div>
</section>
<section class="slide level2">

<div class="centered">
<p><img data-src="figs/aurantiacus/stankowski-color-cline.png" style="width:70.0%" /></p>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:40%;">
<div class="caption">
<p>From <em>Widespread selection and gene flow shape the genomic landscape during a radiation of monkeyflowers</em>, Stankowski, Chase, Fuiten, Rodrigues, Ralph, and Streisfeld; PLoS Bio 2019.</p>
</div>
<div class="centered">
<p><img data-src="figs/sean_stan.jpeg" style="width:45.0%" alt="sean stankowski" /> <img data-src="figs/madeline_chase.jpeg" style="width:45.0%" alt="madeline chase" /> <img data-src="figs/matt_streis.jpeg" style="width:50.0%" alt="matt streisfeld" /></p>
</div>
</div><div class="column" style="width:60%;">
<p><img data-src="figs/aurantiacus/just_phylogeny.png" /></p>
</div>
</div>
</section>
<section class="slide level2">

<p>The data:</p>
<ul>
<li>chromosome-level genome assembly</li>
<li><span class="math inline">\(20\times\)</span> coverage of 8 taxa and outgroup (<em>M.clevelandii</em>)</li>
<li>diversity (<span class="math inline">\(\pi\)</span>), divergence (<span class="math inline">\(d_{xy}\)</span>), and differentiation (<span class="math inline">\(F_{ST}\)</span>) in windows</li>
<li>36 pairwise comparisons among 9 taxa</li>
<li>estimates of recombination rate and gene density from map and annotation</li>
</ul>
</section>
<section class="slide level2">

<div class="centered">
<p><img data-src="figs/aurantiacus/lg3.png" style="width:50.0%" /></p>
</div>
</section>
<section class="slide level2">

<p><span class="math display">\[
\begin{aligned}
\pi &amp;= \text{ (within-pop genetic distance) } \\
d_{xy} &amp;= \text{ (between-pop genetic distance) }
\end{aligned}
\]</span></p>
<div class="centered">
<p><img data-src="figs/fst_tree.png" style="width:60.0%" /></p>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_puniceus_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_puniceus.png" style="width:250.0%" /></p>
</div>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_long-caly_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_long-caly.png" style="width:250.0%" /></p>
</div>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_caly_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_caly.png" style="width:250.0%" /></p>
</div>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_aura_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_aura.png" style="width:250.0%" /></p>
</div>
</div>
</section>
<section id="section-4" class="slide level2">
<h2><!--
---------------------

:::: {.columns}
:::::::: {.column width=80%}

![](figs/aurantiacus/rising_landscapes/divergence_by_node_arid-parv_LG3.png)

::::
:::::::: {.column width=20%}

![](figs/aurantiacus/labeled_phylogeny_arid-parv.png){width="250%"}

::::
::::::::
--></h2>
<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_parv_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_parv.png" style="width:250.0%" /></p>
</div>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_gran_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_gran.png" style="width:250.0%" /></p>
</div>
</div>
</section>
<section class="slide level2">

<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_all_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_all.png" style="width:250.0%" /></p>
</div>
</div>
</section></section>
<section>
<section id="some-questions" class="title-slide slide level1">
<h1>Some questions</h1>

</section>
<section id="ok-then-selection." class="slide level2">
<h2>Ok, then: selection.</h2>
<div class="columns">
<div class="column" style="width:50%;">
<p><strong>But:</strong> what kind of selection?</p>
<ul>
<li>newly adaptive variants?</li>
<li>purifying selection?</li>
<li>local adaptation?</li>
<li>selection for introgression?</li>
</ul>
</div><div class="column" style="width:50%;">
<figure>
<img data-src="figs/aurantiacus/just_map.png" alt="map of mimulus" /><figcaption aria-hidden="true">map of mimulus</figcaption>
</figure>
</div>
</div>
</section>
<section class="slide level2">

<p>To test theories and fit models, we need <em>simulations</em> with realistic</p>
<ol type="1">
<li>population sizes,</li>
<li>genomes,</li>
<li>selective pressures,</li>
<li>histories, and</li>
<li>geography.</li>
</ol>
</section>
<section id="section-5" class="slide level2" data-background-image="figs/oregon_geological_map.png">
<h2 data-background-image="figs/oregon_geological_map.png"></h2>
<!-- Tree sequences -->
</section></section>
<section>
<section id="the-tree-sequence" class="title-slide slide level1">
<h1>The tree sequence</h1>

</section>
<section id="history-is-a-sequence-of-trees" class="slide level2">
<h2>History is a sequence of trees</h2>
<p>For a set of sampled chromosomes, at each position along the genome there is a genealogical tree that says how they are related.</p>
<div class="fragment">
<figure>
<img data-src="figs/example.svg" alt="Trees along a chromosome" /><figcaption aria-hidden="true">Trees along a chromosome</figcaption>
</figure>
</div>
</section>
<section class="slide level2">

<p>The <strong>tree sequence</strong> is a way to describe this, er, sequence of trees.</p>
</section>
<section class="slide level2">

<figure>
<img data-src="figs/ts_ex/tree_sequence_genotypes.png" alt="genotypes" /><figcaption aria-hidden="true">genotypes</figcaption>
</figure>
</section>
<section class="slide level2">

<figure>
<img data-src="figs/ts_ex/tree_sequence_genotype_and_tree.png" alt="genotypes and a tree" /><figcaption aria-hidden="true">genotypes and a tree</figcaption>
</figure>
</section>
<section class="slide level2">

<figure>
<img data-src="figs/ts_ex/tree_sequence_next_genotype_and_tree.png" alt="genotypes and the next tree" /><figcaption aria-hidden="true">genotypes and the next tree</figcaption>
</figure>
</section>
<section class="slide level2">

<p><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004842">Kelleher, Etheridge, and McVean</a> introduced the <strong>tree sequence</strong> data structure for a fast coalescent simulator, <a href="https://github.com/jeromekelleher/msprime">msprime</a>.</p>
<ul>
<li><p>stores sequence <em>and</em> genealogical data <strong>very</strong> efficiently</p></li>
<li><p>tree-based sequence storage closely related to haplotype-matching compression</p></li>
<li><p><a href="https://github.com/tskit-dev/tskit"><code>tskit</code></a> : python/C tools</p></li>
</ul>
<div class="columns">
<div class="column" style="width:50%;">
<figure>
<img data-src="figs/tskit_logo.png" style="width:80.0%" alt="tskit logo" /><figcaption aria-hidden="true">tskit logo</figcaption>
</figure>
</div><div class="column" style="width:50%;">
<div class="floatright">
<figure>
<img data-src="figs/jerome.jpeg" style="width:50.0%" alt="jerome kelleher" /><figcaption aria-hidden="true">jerome kelleher</figcaption>
</figure>
<div class="caption">
<p>jerome kelleher</p>
</div>
</div>
</div>
</div>
</section>
<section id="file-sizes" class="slide level2">
<h2>File sizes</h2>
<div class="centered">
<figure>
<img data-src="figs/tsinfer_sizes.png" style="width:90.0%" alt="file sizes" /><figcaption aria-hidden="true">file sizes</figcaption>
</figure>
</div>
<div class="caption">
<p>from <a href="https://www.nature.com/articles/s41588-019-0483-y">Kelleher et al 2018, <em>Inferring whole-genome histories in large population datasets</em></a>, Nature Genetics</p>
</div>
<!-- Estimated sizes of files required to store the genetic variation data for a
simulated human-like chromosome (100 megabases) for up to 10 billion haploid
(5 billion diploid) samples. Simulations were run for 10 1 up to 10 7 haplotypes
using msprime [Kelleher et al., 2016], and the sizes of the resulting files plotted
(points). -->
</section>
<section id="computation-run-time" class="slide level2">
<h2>Computation run time</h2>
<div class="centered">
<figure>
<img data-src="figs/treestats/benchmarks_without_copy_longer_genome.png" style="width:70.0%" alt="efficiency of treestat computation" /><figcaption aria-hidden="true">efficiency of treestat computation</figcaption>
</figure>
</div>
<div class="caption">
<p>from Ralph, Thornton and Kelleher 2019, <a href="https://doi.org/10.1534/genetics.120.303253">Efficiently summarizing relationships in large samples</a></p>
</div>
</section></section>
<section>
<section id="application-to-genomic-simulations" class="title-slide slide level1">
<h1>Application to genomic simulations</h1>

</section>
<section id="the-main-idea" class="slide level2">
<h2>The main idea</h2>
<p>If we <em>record the tree sequence</em> that relates everyone to everyone else,</p>
<p>after the simulation is over we can put neutral mutations down on the trees.</p>
<div class="fragment">
<p>Since neutral mutations don’t affect demography,</p>
<p>this is <em>equivalent</em> to having kept track of them throughout.</p>
</div>
<div class="fragment">
<div class="columns">
<div class="column" style="width:50%;">
<div class="caption">
<p>From Kelleher, Thornton, Ashander, and Ralph 2018, <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006581">Efficient pedigree recording for fast population genetics simulation</a>.</p>
<p>and Haller, Galloway, Kelleher, Messer, and Ralph 2018, <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/1755-0998.12968"><em>Tree‐sequence recording in SLiM opens new horizons for forward‐time simulation of whole genomes</em></a></p>
</div>
</div><div class="column" style="width:50%;">
<div class="floatright">
<p><img data-src="figs/jared.jpeg" style="width:35.0%" alt="jared galloway" /> <img data-src="figs/jaime.jpg" style="width:30.0%" alt="jaime ashander" /> <img data-src="figs/ben-haller.jpg" style="width:30.0%" alt="ben haller" /></p>
</div>
</div>
</div>
</div>
</section>
<section class="slide level2">

<p>This means recording the entire genetic history of <strong>everyone</strong> in the population, <strong>ever</strong>.</p>
<div class="fragment">
<p>It is <em>not</em> clear this is a good idea.</p>
</div>
<div class="fragment">
<p>But, with a few tricks…</p>
</div>
</section>
<section id="a-100x-speedup" class="slide level2">
<h2>A 100x speedup!</h2>
<div class="columns">
<div class="column" style="width:40%;">
<div class="centered">
<figure>
<img data-src="figs/slim_logo.png" style="width:100.0%" alt="SLiM logo" /><figcaption aria-hidden="true">SLiM logo</figcaption>
</figure>
</div>
<!--
{data-background-image="figs/slim_timing_no_msprime.png" data-background-size=40% data-background-position=left}
-->
</div><div class="column" style="width:60%;">
<div class="floatright">
<p><img data-src="figs/slim_timing_no_msprime.png" style="width:100.0%" /></p>
</div>
</div>
</div>
</section>
<section id="what-else-can-you-do-with-tree-sequences" class="slide level2">
<h2>What else can you do with tree sequences?</h2>
<ul>
<li class="fragment">recorded pedigree and migration history</li>
<li class="fragment">true ancestry assignment</li>
<li class="fragment"><em>recapitation</em>: fast, post-hoc initialization with coalescent simulation</li>
<li class="fragment">fast, convenient computation</li>
</ul>
</section>
<section class="slide level2">

<p><img data-src="figs/run_1.5_50.0_5.0_30_23.line_flux.png" /></p>
</section>
<section class="slide level2">

<p><strong>For example:</strong></p>
<ul>
<li>genome as human chr7 (<span class="math inline">\(1.54 \times 10^8\)</span>bp)</li>
<li><span class="math inline">\(\approx\)</span> 10,000 diploids</li>
<li>500,000 overlapping generations</li>
<li>continuous, square habitat</li>
<li>selected mutations at rate <span class="math inline">\(10^{-10}\)</span></li>
<li>neutral mutations added afterwards</li>
</ul>
<p><em>Runtime:</em> 8 hours</p>
</section></section>
<section>
<section id="back-to-mimulus" class="title-slide slide level1">
<h1>Back to Mimulus</h1>

</section>
<section id="the-data" class="slide level2">
<h2>The data</h2>
<div class="columns">
<div class="column" style="width:80%;">
<p><img data-src="figs/aurantiacus/rising_landscapes/divergence_by_node_puniceus_LG3.png" /></p>
</div><div class="column" style="width:20%;">
<p><img data-src="figs/aurantiacus/labeled_phylogeny_puniceus.png" style="width:250.0%" /></p>
</div>
</div>
</section>
<section id="simulations" class="slide level2">
<h2>Simulations</h2>
<div class="columns">
<div class="column" style="width:70%;">
<ul>
<li><p><span class="math inline">\(N=10,000\)</span> diploids</p></li>
<li><p>burn-in for <span class="math inline">\(10N\)</span> generations</p></li>
<li><p>population split, with either:</p>
<ul>
<li>neutral</li>
<li>background selection</li>
<li>selection against introgressed alleles</li>
<li>positive selection</li>
<li>local adaptation</li>
</ul></li>
</ul>
</div><div class="column" style="width:30%;">
<div class="flushright">
<p><img data-src="figs/murillo.jpeg" /></p>
<div class="caption">
<p>Murillo Rodrigues</p>
</div>
</div>
</div>
</div>
</section>
<section class="slide level2">

<!--
Fig 7. Genomic landscapes simulated under different divergence histories.
Each row of plots shows patterns of within- and between-population variation (π, dxy, and FST) across the chromosome (500-kb windows) at 5 time points (N generations, where N = 10,000) during one of the scenarios The selection parameter (Ns, where s = Ns/N), proportion of deleterious (−) and positive mutations (+), and number of migrants per generation (Nm; 0 unless stated) for these simulations are as follows: (i) neutral divergence (no selection), (ii) BGS (−Ns = 100; −prop = 0.1), (iii) BDMI (−Ns = 100, −prop = 0.05, Nm = 0.1), (iv) positive selection (+Ns = 100, +prop = 0.001), (v) BGS and positive selection (−Ns = 100, −prop = 0.1; +Ns = 100, +prop = 0.005), and (vi) local adaptation (+Ns = 100, +prop = 0.001, Nm = 0.1). The gray boxes in the first column show the areas of the chromosome that are experiencing selection, while the white central area evolves neutrally. Note that π (in populations a and b) and dxy have been mean centered so they can be viewed on the same scale. Uncentered values and additional simulations with different parameter combinations and more time points can be found in S13 Fig. BDMI, Bateson-Dobzhansky-Muller incompatibility; BGS, background selection.

![](figs/aurantiacus/sim_results.png)
-->
<p><img data-src="figs/sim_mimulus_landscapes.svg" style="width:100.0%" /></p>
<div class="flushright">
<div class="caption">
<p>From <a href="https://doi.org/10.1371/journal.pbio.3000391">Widespread selection and gene flow shape the genomic landscape during a radiation of monkeyflowers</a>, Stankowski, Chase, Fuiten, Rodrigues, Ralph, and Streisfeld; PLoS Bio 2019.</p>
</div>
</div>
</section>
<section class="slide level2">

<p>Conclusions:</p>
<ul>
<li><strike>neutral</strike></li>
<li><strike>background selection</strike></li>
<li><strike>selection against introgressed alleles</strike></li>
<li>positive selection</li>
<li>local adaptation</li>
</ul>
</section></section>
<section>
<section id="wrap-up" class="title-slide slide level1">
<h1>Wrap-up</h1>

</section>
<section id="bigger-picture" class="slide level2">
<h2>Bigger picture</h2>
<p>How strongly does selection enhance or constrain genetic variation?</p>
<div class="fragment">
<p>How much genetic variation is locally adaptive?</p>
</div>
<div class="fragment">
<p>How will populations respond to changes in the future?</p>
</div>
</section>
<section id="other-uses-for-population-simulation" class="slide level2">
<h2>Other uses for population simulation</h2>
<div class="columns">
<div class="column" style="width:50%;">
<ul>
<li><p>train inference methods</p></li>
<li><p>predict management outcomes</p></li>
<li><p>do power analyses</p></li>
<li><p>develop intuition</p></li>
</ul>
</div><div class="column" style="width:50%;">
<div class="centered">
<figure>
<img data-src="figs/tskit_logo.png" style="width:60.0%" alt="tskit logo" /><figcaption aria-hidden="true">tskit logo</figcaption>
</figure>
<figure>
<img data-src="figs/slim_logo.png" style="width:100.0%" alt="SLiM logo" /><figcaption aria-hidden="true">SLiM logo</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="section-6" class="slide level2" data-background-image="figs/slim_screenshot.png" data-background-position="center" data-background-size="100%">
<h2 data-background-image="figs/slim_screenshot.png" data-background-position="center" data-background-size="100%"></h2>
</section>
<section id="section-7" class="slide level2">
<h2></h2>
<div class="columns">
<div class="column" style="width:50%;">
<p>The Co-Lab:</p>
<ul>
<li>Andy Kern</li>
<li>Matt Lukac</li>
<li>Murillo Rodrigues</li>
<li>Jared Galloway</li>
<li>CJ Battey</li>
</ul>
<p>Funding:</p>
<ul>
<li>NIH NIGMS</li>
<li>NSF DBI</li>
<li>Sloan foundation</li>
<li>UO Data Science</li>
</ul>
</div><div class="column" style="width:50%;">
<div style="font-size: 85%; margin-top: -40px;">
<p>Other collaborators:</p>
<ul>
<li>Jerome Kelleher</li>
<li>Ben Haller</li>
<li>Ben Jeffery</li>
<li>Georgia Tsambos</li>
<li>Jaime Ashander</li>
<li>Gideon Bradburd</li>
<li>Madeline Chase</li>
<li>Bill Cresko</li>
<li>Alison Etheridge</li>
<li>Evan McCartney-Melstad</li>
<li>Brad Shaffer</li>
<li>Sean Stankowski</li>
<li>Matt Streisfeld</li>
<li>Anastasia Teterina</li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-8" class="slide level2" data-background-image="figs/guillemots_thanks.png" data-background-position="center" data-background-size="50%">
<h2 data-background-image="figs/guillemots_thanks.png" data-background-position="center" data-background-size="50%"></h2>
</section></section>
    </div>
  </div>

  <script src="https://unpkg.com/reveal.js@^4//dist/reveal.js"></script>

  // reveal.js plugins
  <script src="https://unpkg.com/reveal.js@^4//plugin/notes/notes.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/search/search.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/zoom/zoom.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/math/math.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,
        // Transition style
        transition: 'none', // none/fade/slide/convex/concave/zoom
        math: {
          mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js',
          config: 'TeX-AMS_HTML-full',
          tex2jax: {
            inlineMath: [['\\(','\\)']],
            displayMath: [['\\[','\\]']],
            balanceBraces: true,
            processEscapes: false,
            processRefs: true,
            processEnvironments: true,
            preview: 'TeX',
            skipTags: ['script','noscript','style','textarea','pre','code'],
            ignoreClass: 'tex2jax_ignore',
            processClass: 'tex2jax_process'
          },
        },

        // reveal.js plugins
        plugins: [
          RevealMath,
          RevealNotes,
          RevealSearch,
          RevealZoom
        ]
      });
    </script>
    </body>
</html>
